src/lib/disk_ops.rs

```diff
@@ -1587,6 +1587,7 @@ mod tests {
     #[test]
     fn should_store_all_dbs() {
         let dbs = create_test_dbs();
+        // dbs.is_oplog_valid.store(false, Ordering::Relaxed);
```

Comment:
Cada código comentado aleatóriamente, um panda morre.
Ass: Uncle Bob

Reply:
Ship it!

Reply:
só de estar em rust, já merece deslike

Reply:
é

Reply:
queria eu ter um desse no meu perfil

Reply:
lgtm

Reply:
Projeto ficou brabo mano!